daemon/datapath.go

```diff
@@ -335,3 +337,39 @@ func (d *Daemon) writeNetdevHeader(dir string) error {
 	}
 	return nil
 }
+
+// EndpointMapManager is a wrapper around an endpointmanager as well as the
+// filesystem for removing maps related to endpoints from the filesystem.
+type EndpointMapManager struct {
+	endpointManager *endpointmanager.EndpointManager
+}
+
+// EndpointExists returns whether the endpoint with the specified ID is managed
+// by this Cilium instance.
+func (e *EndpointMapManager) EndpointExists(endpointID uint16) bool {
```

Comment:
It seems reasonable to add `EndpointExists` and `HasGlobalCT` to endpoint manager.

Reply:
We has this for `HasGlobalCT`, but not for `EndpointExists`. I'll look into seeing how I can play around with the code to make this less boilerplate-y.

Reply:
I've added `EndpointExists` to the `EndpointManager`, and now the `EndpointMapManager` is a composition of `*endpointmanager.EndpointManager`.